models/User.go

```diff
@@ -702,15 +709,15 @@ func GetUserCopy(id uint32) (User, error) {
 	return *u.user, nil
 }
 
-// SubtractOrderFee subtracts cash for users account
-func SubtractOrderFee(user *User, orderFee uint64, tx *gorm.DB) error {
+// SubtractUserCash subtracts cash for users account
+func SubtractUserCash(user *User, fee uint64, tx *gorm.DB) error {
```

Comment:
Does this need to be exported?

Reply:
As of now, the only places using it are from within the same file, so no. Also, wouldn't it be more ideal to have this as a member function of the User class rather than taking User as a parameter?